demo/src/components/Cart/Cart.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Container, Col, Row, Form} from 'react-bootstrap';
+import './Cart.css';
+import {setQuantity} from '../../store/StoreHelpers';
+import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates a component describing a shopping Cart.
+ * @param {!Object} state The site state
+ * @param {function} setQuantity function to modify quantity of items
+ *      in the global state
+ * @return {!JSX} The component.
+ */
+const CartBase = function Cart({items, setQuantity}) {
+  const itemsRender = [];
+
+  /*
+   * Create the content of the cart display, with one row per item.
```

Comment:
Please use single line comment // for one-line comments

Reply:
Fixed